packages/critters/src/index.js

```diff
@@ -477,7 +477,14 @@ export default class Critters {
       ast,
       markOnly((rule) => {
         if (rule.type === 'comment') {
-          const comment = rule.text.trim();
+          let comment = rule.text;
```

Comment:
NIT: The following should be more performant.

```js
const crittersComment = rule.text.match( /^!? ?critters\:(.*)/);
const command = crittersComment && crittersComment[1];
```

Removes the need for multiple `.startsWith`, `.trim` `.slice`, and `.replace` and also reduces memory allocation.